_delphi_utils_python/tests/validator/test_static.py

```diff
@@ -302,6 +316,31 @@ def test_invalid_geo_id_value_nation(self):
         assert len(report.raised_errors) == 1
         assert report.raised_errors[0].check_name == "check_bad_geo_id_value"
 
+    def test_additional_valid_geo_ids(self):
+        params = self.params.copy()
+        params["static"] = {
+            "additional_valid_geo_values": {
+                    "state": ["confusion"],
+                    "county": ["left", "right"]
```

Comment:
nit: could you name these something like "additional_county1" so the below dataframe values are fairly clear

Reply:
done

Reply:
update state name too? otherwise lgtm